src/framework/uicomponents/uicomponentstypes.h

```diff
@@ -57,7 +57,8 @@ struct MenuItem : public actions::ActionItem
             { "icon", static_cast<int>(iconCode) },
             { "enabled", enabled },
             { "checked", checked },
-            { "subitems", _subitems }
+            { "subitems", _subitems },
+            { "data", data }
```

Comment:
I think, better to rename data to params or args

Reply:
thanks, renamed to args